### DISCUSSION TITLE
Container Registry packages not showing

### DISCUSSION THREAD

**Discussion by tyrads** [community/community#36774] [Packages]
I am pushing and pulling docker images from docker container registry but they are not showing in  either packages tab or section under settings.

**Comment by mherger** [comment]
Same here: I've successfully pushed two images and use them in a workflow. But I can't see it anywhere among my packages or wherever they should show up. I'm part of a corporate account ("Enterprise"?), if that matters.

Is there any UI to ghcr.io?

**Reply by tyrads** [reply]
<img width="224" alt="image" src="https://user-images.githubusercontent.com/108784253/201257855-dddb3bb4-48c3-457c-841f-cf99ca035279.png">
Every repository has a packages section, but nothing showing up.
